### PR TITLE
feat(rdb): read replica MTA-2185

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**
Added:
How to create read replicas
How to manage read replicas

### CODE REVIEW

File: managed-databases/postgresql-and-mysql/how-to/manage-read-replica.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## How to resync a read replica
+
+When you resync a read replica, first it is reset, and then its data is resynchronized from the primary node. Your read replica will be unavailable during the resync process. The duration of this process is proportional to your Database Instance size. The configured endpoints will not change.
+
+1. Click **PostgreSQL and MySQL** under **Managed Databases** on the side menu. A list of your Database Instances displays.
+2. Click the database name or <Icon name="more" /> > **More info** to access the Database Instance information page.
+3. Scroll down to the **Read Replica** section. 
+4. Click <Icon name="more" /> > **Resync** next to the name of the read replica you want to resync. A pop-up displays.
+
+  <Lightbox src="scaleway-read-replica-resync.webp" alt="A Pop-up displays the following message: Resync data from primary node. Are you sure you want to reset and resync the data of 2c241214-030e-4900-a993-579ba6fa2da7 from the main node? Your read replica will be unavailable during the resync process. The duration of this process is proportional to your Database Instance size." />
```
Comment (bene2k1) [inline managed-databases/postgresql-and-mysql/how-to/manage-read-replica.mdx:30]:
```suggestion
  <Lightbox src="scaleway-read-replica-resync.webp" alt="A pop-up displays the following message: Resync data from primary node. Are you sure you want to reset and resync the data of 2c241214-030e-4900-a993-579ba6fa2da7 from the main node? Your read replica will be unavailable during the resync process. The duration of this process is proportional to your Database Instance size." />
```

**Review by bene2k1 — APPROVED** [review]

File: managed-databases/postgresql-and-mysql/how-to/create-read-replica.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  h1: How to create read replicas
+  paragraph: This page explains how to create read replicas
+dates:
+  validation: 2022-08-31
+  posted: 2019-09-21
+---
+
+A read replica is a live copy of a Database Instance that behaves like an Instance, but that only allows read-only connections. The replica mirrors the data of the primary Database Instance and any changes made are synchronized and reflected in the replica.
+
+Read replica allows you to scale your Database Instance for read-heavy database workloads. They can also be used for business intelligence workloads.
```
Comment (RoRoJ) [inline managed-databases/postgresql-and-mysql/how-to/create-read-replica.mdx:15]:
```suggestion
Read replicas allow you to scale your Database Instance for read-heavy database workloads. They can also be used for business intelligence workloads. 
```

File: managed-databases/postgresql-and-mysql/concepts.mdx [hunk 3]
```diff
@@ -85,6 +85,14 @@ Compared to traditional database management, which requires customers to provisi
 
 <Concept>
 
+## Read replica
+
+A read replica is a live copy of a Database Instance that behaves like an Instance, but that only allows read-only connections. The replica mirrors the data of the primary Database Instance and any changes made are synchronized and reflected in the replica.
```
Comment (nzerhouni) [inline managed-databases/postgresql-and-mysql/concepts.mdx:90]:
```suggestion
A read replica is a live copy of a Database Instance that behaves like an Instance, but that only allows read-only connections. The replica mirrors the data of the primary Database node and any changes made are synchronized and reflected in the replica.
```

File: managed-databases/postgresql-and-mysql/how-to/create-read-replica.mdx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  description: This page explains how to create read replicas
+content:
+  h1: How to create read replicas
+  paragraph: This page explains how to create read replicas
+dates:
+  validation: 2022-08-31
+  posted: 2019-09-21
+---
+
+A read replica is a live copy of a Database Instance that behaves like an Instance, but that only allows read-only connections. The replica mirrors the data of the primary Database Instance and any changes made are synchronized and reflected in the replica.
```
Comment (nzerhouni) [inline managed-databases/postgresql-and-mysql/how-to/create-read-replica.mdx:13]:
```suggestion
A read replica is a live copy of a Database Instance that behaves like an Instance, but that only allows read-only connections. The replica mirrors the data of the primary Database node and any changes made are synchronized and reflected in the replica.
```

File: managed-databases/postgresql-and-mysql/how-to/create-read-replica.mdx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+## How to create a read replica
+
+1. Click **PostgreSQL and MySQL** under **Managed Databases** on the side menu. A list of your Database Instances displays.
+2. Click the database name or <Icon name="more" /> > **More info** to access the Database Instance information page.
+3. Scroll down to the **Read Replica** section. 
+4. Click **Create Read Replica**. A pop-up displays.
+5. Select the network type. Two types are available:
+
+    - **Public Network** - Resources in a public network are publicly visible by default. The default Access Control List (ACL) settings allow anybody to connect to resources in your network. You can restrict access by configuring ACLs.
```
Comment (nzerhouni) [inline managed-databases/postgresql-and-mysql/how-to/create-read-replica.mdx:32]:
```suggestion
    - **Public Network** - Resources in a public network are publicly visible by default. The default Access Control List (ACL) settings allow anybody to connect to resources in your network. You can restrict access by configuring ACLs under the Allowed IPs subtab.
```

File: managed-databases/postgresql-and-mysql/how-to/create-read-replica.mdx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    - **Private Networks** - Private Networks allow you to create virtual Private Networks for your Databases, enabling them to communicate in an isolated network without the need for a public IP address. If you select this option, follow these additional steps:
+      - Select an existing Private Network to attach to your read replica.
+      - Configure your IP address and its corresponding mask. 
+
+  <Message type="note">
+
+  Read replicas can have at most one direct access and one Private Network endpoint.
+
```
Comment (nzerhouni) [inline managed-databases/postgresql-and-mysql/how-to/create-read-replica.mdx:40]: Maybe we can add a N.B. saying that if the user choses public network at this level, he can still attach a private network to his read replica afterwards. 
If he choses private network, he will not be able to add a public network to the same read replica.

File: managed-databases/postgresql-and-mysql/how-to/create-read-replica.mdx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+  </Message>
+6. Click **Create read replica**. A list of your read replicas displays.
+
+<Message type="note">
+
+  The read replica creation process may take a couple minutes.
```
Comment (nzerhouni) [inline managed-databases/postgresql-and-mysql/how-to/create-read-replica.mdx:46]: depending on the volume of data that needs to be replicated

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**